desktop-app/app/menu.js

```diff
@@ -93,6 +94,21 @@ export default class MenuBuilder {
           });
         },
       },
+      {
+        label: 'Check for Updates...',
+        click() {
+          autoUpdater.checkForUpdatesAndNotify().then(r => {
+            if (r == null || r.updateInfo == null || r.updateInfo.version === pkg.version) {
```

Comment:
Maybe in the `else` we should show an appropriate message saying "Update in progress" or something like that?

Reply:
Yes we could, I just followed the `vscode` behavior: if there is no update a dialog is shown and if there is an update a notification is shown. In `vscode` the notification is not a system notification but I though that it would be fine, because the notification shows a very descriptive message. But I'll go with whatever you want @manojVivek  , just tell me your idea